prisma/migrations/20210428060853_user_username_email_required_unique/migration.sql

```diff
@@ -0,0 +1,20 @@
+/*
+  Warnings:
+
+  - A unique constraint covering the columns `[username]` on the table `users` will be added. If there are existing duplicate values, this will fail.
+  - A unique constraint covering the columns `[email]` on the table `users` will be added. If there are existing duplicate values, this will fail.
+  - Made the column `name` on table `users` required. This step will fail if there are existing NULL values in that column.
+  - Made the column `username` on table `users` required. This step will fail if there are existing NULL values in that column.
+  - Made the column `email` on table `users` required. This step will fail if there are existing NULL values in that column.
+
+*/
+-- AlterTable
+ALTER TABLE "users" ALTER COLUMN "name" SET NOT NULL,
+ALTER COLUMN "username" SET NOT NULL,
+ALTER COLUMN "email" SET NOT NULL;
+
+-- CreateIndex
+CREATE UNIQUE INDEX "users.username_unique" ON "users"("username");
+
+-- CreateIndex
+CREATE UNIQUE INDEX "users.email_unique" ON "users"("email");
```

Comment:
I think it's better if we create these unique indexes on a later migration until we figure out a way to do it safely.

For now, let's just make the columns not null to solve the typescript complaints.

```suggestion
```